lib/rules/web/admin_console/4.6/overview.xyaml

```diff
@@ -216,4 +216,63 @@ group_by_label:
     dropdown_menu_item: <label>
   action: click_groupby_dropdown_button
   action: click_dropdown_menu_item
-
+click_workloads_tab:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: click
+send_key_to_workloads:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: send_keys <key>
+check_workload_selected:
+  elements:
+  - selector:
+      xpath: //div[@class='list-group-item project-overview__item project-overview__item--<workload> project-overview__item--selected']
```

Comment:
In `check_workload_selected` we can check one workload with <workload_name> is selected 
```
check_workload_selected:
  elements:
  - selector:
        xpath: //li[contains(@class,'pf-c-data-list__item') and contains(@class,'pf-m-selected')]//div[contains(.,'console')]
```

Reply:
In this way, we don't need rule `check_workload_name`

Reply:
@yapei Made the change to use the above but since I'm validating the details in the side bar also need the check details step.